### PR TITLE
 FAS: Routing slip linkage(How its done) | designs

### PR DESCRIPTION
**Pull request by shabeeb-aot**
*Issue #:* /bcgov/entity#8530

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/ViewRoutingSlip/LinkRoutingSlip.vue [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    </header>
+    <v-card class="pl-5 py-2 small-text-input">
+      <v-card-text>
+        <v-row v-if="alreadyLinked">
+          <v-col class="col-6 col-sm-8 font-weight-bold">
+            This routing slip has been linked to:
+          </v-col>
+          <v-col cols="12">
+            <LinkedRSDetails
+              createdDate="May 05, 2021"
```
Comment (shabeeb-aot) [inline src/components/ViewRoutingSlip/LinkRoutingSlip.vue:15]: sample data untill next PR

**Review by saravanpa-aot — APPROVED** [review]

File: src/components/ViewRoutingSlip/LinkedRSDetails.vue [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<template>
```
Comment (sumesh-aot) [inline src/components/ViewRoutingSlip/LinkedRSDetails.vue:1]: We may need some consistency in file naming, either use `RS` across all files, or use `RoutingSlip`

**Review by sumesh-aot — APPROVED** [review]
Please see my comment

File: src/components/ViewRoutingSlip/LinkedRSDetails.vue [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import commonUtil from '@/util/common-util'
+
+@Component({})
+export default class LinkedRSDetails extends Vue {
+  public colors = commonUtil.statusListColor
+  public formatDisplayDate = commonUtil.formatDisplayDate
+  @Prop({ default: '' }) private SiNumber: string
+  @Prop({ default: '' }) private RSNumber: string
```
Comment (karthik-aot) [inline src/components/ViewRoutingSlip/LinkedRSDetails.vue:28]: can we expand here as well ? SInumber and RSnumber for readability?

File: src/components/ViewRoutingSlip/LinkRoutingSlip.vue [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+  text-transform: capitalize;
+}
+.row + .row {
+  margin-top: 7px !important;
+}
+.status-list {
+  max-width: 400px;
+}
+.linked-rs-info {
+  background-color: #f1f3f5;
```
Comment (karthik-aot) [inline src/components/ViewRoutingSlip/LinkRoutingSlip.vue:151]: nice :)

File: src/components/ViewRoutingSlip/LinkedRSDetails.vue [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  </div>
+</template>
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import commonUtil from '@/util/common-util'
+
+@Component({})
+export default class LinkedRSDetails extends Vue {
+  public colors = commonUtil.statusListColor
+  public formatDisplayDate = commonUtil.formatDisplayDate
```
Comment (karthik-aot) [inline src/components/ViewRoutingSlip/LinkedRSDetails.vue:26]: Please re arrange code. Prop first and then commonutil :)

**Review by karthik-aot — APPROVED** [review]

**Pull request merged**